teflo_init/provision/README.rst

```diff
@@ -0,0 +1,51 @@
+Static
```

Comment:
What does  Static  mean here , should the heading be Provision_Example ?

Reply:
fixed